Dockerfile

```diff
@@ -58,8 +55,10 @@ CMD [ "--host=0.0.0.0" ]
 # Create an image used for running the app in a production environment.
 FROM base-with-app-code as prod
 
+RUN export FLASK_ENV=production
+
 # Define commands to be run when container is started.
-CMD [ "poetry", "run", "gunicorn", "--bind=0.0.0.0:5000", "--chdir", "./application", "app:create_app()" ]
+CMD poetry run gunicorn --bind=0.0.0.0:$PORT --chdir ./application 'app:create_app()'
```

Comment:
Note that this means you are insisting that a PORT environment variable is set - this means your local docker instructions for prod will only work if a PORT variable is set (and so following the README doesn't necessarily work any more)

We could fix that by setting a default environment variable in the Dockerfile (e.g. `ENV PORT=5000`) or by asking the user to pass one through in their `.env`